concent_api/core/message_handlers.py

```diff
@@ -954,6 +958,58 @@ def store_message(
     return stored_message
 
 
+def handle_send_subtask_results_verify(
+    request,
+    subtask_results_verify: message.concents.SubtaskResultsVerify
+):
+    report_computed_task = subtask_results_verify.subtask_results_rejected.report_computed_task
+    compute_task_def = report_computed_task.task_to_compute.compute_task_def
+    validate_id_value(
+        compute_task_def['subtask_id'],
+        'subtask_id'
+    )
+    client_public_key = decode_client_public_key(request)
+    other_party_public_key = decode_other_party_public_key(request)
+
+    if Subtask.objects.filter(
+        Q(subtask_id=compute_task_def['subtask_id']),
```

Comment:
I think `Q` is not needed in this line. Generally this can be rewritten to:
```
Subtask.objects.filter(
    subtask_id=compute_task_def['subtask_id'],
    Q(state__in=[
        Subtask.SubtaskState.VERIFICATION_FILE_TRANSFER.name,
        Subtask.SubtaskState.ADDITIONAL_VERIFICATION.name
    ]
).exists()
```

Reply:
done